### ISSUE TITLE
micro win ui not updated after unpack of iso

### ISSUE THREAD

**Issue by nirmal0001** [ChrisTitusTech/winutil#2930]
## Describe the bug
ui should be updated after iso unpack but not happening tried it on 2 device

## To Reproduce
1. Go to 'microwin.'
2. Click on 'select iso.'
3. select iso
4. wait
5. no update on screen when terminal say check ui for further steps

## Expected behavior
show option for iso 

## Additional context
oscdimg.exe on system: True
File path C:\Users\nimmadev\Downloads\Win11_23H2_English_x64v2.iso
You have enough space for this operation.
Mounting Iso. Please wait.
Done mounting Iso MSFT_DiskImage (ImagePath = "C:\Users\nimmadev\Downloads\Win11_23H2_..., StorageType = 1)
Iso mounted to 'E'
Setting up mount dir and scratch dirs
Mounting Iso. Please wait.
Done setting up mount dir and scratch dirs
Scratch dir is C:\Users\nimmadev\AppData\Local\Temp\MicrowinScratch_20241012_232339_2928
Image dir is C:\Users\nimmadev\AppData\Local\Temp\Microwin_20241012_232339_2928
Copying Windows image. This will take awhile, please don't use UI or cancel this step!
Copy 1031 file(s) from E: to C:\Users\nimmadev\AppData\Local\Temp\Microwin_20241012_232339_2928
Copy complete! Total Time: 0m23s
Getting image information C:\Users\nimmadev\AppData\Local\Temp\Microwin_20241012_232339_2928\sources\install.wim
Dismounting bad image...
Done reading and unpacking ISO

*********************************
Check the UI for further steps!!!

**Comment by blusewill** [comment]
Seems like you got the bad copy of the Windows 11 Images.
Probably Download it again from the Microsoft Website and try it again.

**Comment by MyDrift-user** [comment]
Hello @nirmal0001 

Is the UI interactable, meaning can you resize the window and click stuff with everything else working?

If yes are you aware how the UI is updated, meaning where you find the further steps to proceed?

**Comment by nirmal0001** [comment]
The issue was iso but iso was working when using directly
So I didn't doubt that
But after redownloading everything worked

Still would be great if there was an indicator like iso is not good or something


@MyDrift-user it stayed same state when selecting iso and yes everything was functional once iso is extracted just no new UI 
I could still do anything else

**Comment by MyDrift-user** [comment]
Just noticed the error in the terminal, that would be:

```

Getting image information C:\Users\nimmadev\AppData\Local\Temp\Microwin_20241012_232339_2928\sources\install.wim
Dismounting bad image...

```

Sorry for not noticing sooner

**Comment by nirmal0001** [comment]
Yeah just noticed but the last line is more focused which says check the UI! so didn't notice the error 

Should I close the issue
